GoogleDataTransport/GDTCORLibrary/GDTCORReachability.m

```diff
@@ -114,10 +114,12 @@ - (void)setCallbackFlags:(GDTCORNetworkReachabilityFlags)flags {
 
 @end
 
+#pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wunused-function"
 static void GDTCORReachabilityCallback(GDTCORNetworkReachabilityRef reachability,
                                        GDTCORNetworkReachabilityFlags flags,
                                        void *info) {
   GDTCORLogDebug("Reachability changed, new flags: %d", flags);
   [[GDTCORReachability sharedInstance] setCallbackFlags:flags];
 }
+#pragma clang diagnostic pop
```

Comment:
put this under line 121 `void *info) {`

We only want to disable the warning for the lines necessary

Reply:
Fixed